demo/lightning/image-segmentation/README.md

```diff
@@ -0,0 +1,167 @@
+# Image Segmentation Demo Code
+The code examples in this directory demonstrate how Dataflux can be used for image segmentation training alongside PyTorch Lightning. The image segmentation workload implemented here works on the KiTS 19 dataset which has `210` images and their corresponding labels. The images and their labels are stored in separate directories.
```

Comment:
i would say "which contains 210 images" for clarity.

Reply:
Done.